test/framework/client/collect.go

```diff
@@ -25,33 +26,36 @@ import (
 )
 
 type CollectResponsesOpts struct {
-	numberOfRequests      int
-	maxConcurrentRequests int
-	maxTime               int
-	verbose               bool
-	cacert                string
-	URL                   string
-	Method                string
-	Headers               map[string]string
+	numberOfRequests          int
+	maxConcurrentRequests     int
+	maxConcurrentRequestDelay int
+	maxTime                   int
+	verbose                   bool
+	cacert                    string
+	URL                       string
+	Method                    string
+	Headers                   map[string]string
 
 	Flags        []string
 	ShellEscaped func(string) string
 
 	namespace   string
 	application string
 
-	withoutRetries bool
+	withoutRetries                bool
+	withoutConcurrentRequestDelay bool
 }
 
 func DefaultCollectResponsesOpts() CollectResponsesOpts {
 	return CollectResponsesOpts{
-		numberOfRequests:      10,
-		maxConcurrentRequests: 10,
-		maxTime:               5,
-		verbose:               false,
-		Method:                "GET",
-		Headers:               map[string]string{},
-		ShellEscaped:          utils.ShellEscape,
+		numberOfRequests:          10,
+		maxConcurrentRequests:     10,
```

Comment:
Should we reduce this? like 5 or something like this?

Reply:
We can do it because it produces a bit of load on Envoy.